### ISSUE TITLE
[Accessibility]: Code monitoring create/edit: Closing a trigger/action card doesn't return focus to the right element to open it again

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#35729]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34195

### Problem description

Closing the trigger or action cards with either the continue or cancel buttons will close the card but leave nothing focused on the page.

https://user-images.githubusercontent.com/206864/169371307-c2c09b2b-bdb8-451a-b7d8-672a1febd1bd.mp4

### Expected behavior

Closing the trigger or action cards in the code monitors add/edit page should focus the card again.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
